src/dev/src/app/iconography/icon-selection.demo.html

```diff
@@ -8,7 +8,7 @@
 
     <section>
         <h5>Preview Settings</h5>
-        <div class="row clr-icon-preview-settings">
+        <div class="clr-icon-preview-settings">
```

Comment:
Does it still need to be `clr-row` here?

Reply:
For both of these questions, there were no column cells here and the row class is not necessary.